_includes/footer.html

```diff
@@ -47,10 +47,8 @@ <h2 class="footer-heading">{{ site.title }}</h2>
                    <h4>Contact us</h4>
                    <ul>
                         <li>5th Avenue, New York - United States</li>
-                        <li>Phone: +10 724 1234 567 | Fax: +10 724 1234 567 </li>
+                        <li>Phone: +10 724 1234 567</li>
```

Comment:
In what I just pushed up, there was no phone in the config.yml
I added that a few minutes ago when I thought about it too